Assets/MirageXR/Managers/Activity/ActivityManager.cs

```diff
@@ -1,4 +1,5 @@
 ﻿using i5.Toolkit.Core.VerboseLogging;
+using NSubstitute.Core;
```

Comment:
Is this needed?

Reply:
Artefact of an earlier version. It's no longer present